.env.example

```diff
@@ -3,3 +3,7 @@ VUE_APP_BASE_URL_PLIO=''
 VUE_APP_AWS_SQS_ACCESS_KEY=''
 VUE_APP_AWS_SQS_SECRET_ACCESS_KEY=''
 VUE_APP_BASE_URL_MEET=''
+VUE_APP_AWS_REGION=''
+VUE_APP_AWS_SQS_URL=''
+VUE_APP_BASE_URL_YOUTUBE=''
+VUE_APP_OTP_URL=''
```

Comment:
The youtube and OTP URL's will be added from the PR's. Maybe we can remove it from here.